test/src/test/scala/react/common/CommonSuite.scala

```diff
@@ -0,0 +1,56 @@
+package react.common
+
+import react.common.implicits._
+import cats.syntax.all._
+import scala.scalajs.js
+
+class CommonSuite extends munit.FunSuite with TestUtils {
+  test("merge") {
+    val a: js.Object = js.Dynamic.literal(foo = 42, bar = "foobar")
+    val b            = js.Object()
+    val m            = merge(a, b)
+    assert(a === m)
```

Comment:
Use `assertEquals`? In case of failure it provides more reporting, showing the differences.

Reply:
Actually this is in part a test of `===` while `assertEquals` will use plain equal.